bitnami/wavefront-prometheus-storage-adapter/templates/deployment.yaml

```
@@ -0,0 +1,154 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: wavefront-prometheus-storage-adapter
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "common.names.fullname" . }}
```

Include the `metadata.namespace` on every manifest:
```suggestion
  name: {{ template "common.names.fullname" . }}
  namespace: {{ .Release.Namespace }}
```